lib/backup/fscommon/fscommon.go

```diff
@@ -173,7 +173,6 @@ func removeEmptyDirsInternal(d *os.File) (bool, error) {
 		return false, fmt.Errorf("cannot read directory contents in %q: %w", dir, err)
 	}
 	dirEntries := 0
-	hasFlock := false
```

Comment:
I don't think this code may be safely removed because of https://github.com/VictoriaMetrics/VictoriaMetrics/commit/a7125a5b7b4cd467251e60446efa918c757971b2 . It would be better adding a comment here with the explanation why hasFlock is needed.

Reply:
Fixed.

 I  thought, that it's possible case, when flock.lock file will be removed from rootDir. But with restore.lock it should be ok.